scripts/zones/Nashmau/npcs/Leleroon.lua

```diff
@@ -1,30 +1,28 @@
 -----------------------------------
 -- Area: Nashmau
 -- NPC: Leleroon
--- Standard Info NPC -- Corsair SideQuests
+-- Corsair AF2 and AF3 quests
 -- @pos -14.687 0.000 25.114 53
 -----------------------------------
 package.loaded["scripts/zones/Nashmau/TextIDs"] = nil;
 -----------------------------------
 
-require("scripts/zones/Nashmau/TextIDs");
 require("scripts/globals/settings");
 require("scripts/globals/status");
 require("scripts/globals/quests");
 require("scripts/globals/keyitems");
+require("scripts/zones/Nashmau/TextIDs");
```

Comment:
Aside from terrorizing my OCD by re-ordering the requires\*, all looks good.


###### \* *I compulsively order them widest first, in reality we* ***all*** *should be leaving them alone unless they are actually wrong.*